spago.dhall

```diff
@@ -24,6 +24,7 @@
     , "unsafe-reference"
     , "web-file"
     , "web-uievents"
+    , "debug"
```

Comment:
This should not be added to the `spago.dhall` file.

It's OK if you're using `debug` in development -- you might consider adding this to the test dependencies instead and running your build commands using that config (`spago -x test/test.dhall build`, for example). That way you can freely use `debug` but we don't inadvertently add it as a library dependency.